src/components/Checkout/SocialShare.vue

```diff
@@ -50,18 +50,31 @@
 		</div>
 
 		<div class="share__social social">
-			<button class="social__btn social__btn--facebook">
+			<a
+				class="social__btn social__btn--facebook"
+				:href="facebookShareUrl"
+			>
 				<icon-facebook class="social__icon" />
 				<span>Share</span>
-			</button>
-			<button class="social__btn social__btn--twitter">
+			</a>
+			<a
+				class="social__btn social__btn--twitter"
+				:href="twitterShareUrl"
+				target="_blank"
```

Comment:
Probably a good idea to add rel="noopener" to all of these target="_blank" links.
https://developers.google.com/web/tools/lighthouse/audits/noopener

Reply:
👍  yeah that looks like a good idea